packages/plugin-init/sources/commands/init.ts

```diff
@@ -105,19 +104,18 @@ export default class InitCommand extends BaseCommand {
     manifest.private = this.private;
     manifest.license = configuration.get(`initLicense`);
 
-    await updateAndSave(ppath.join(this.context.cwd, Manifest.fileName), (tracker: Object) => {
-      manifest.exportTo(tracker);
-    });
-
-    const inspectable: any = {};
-    manifest.exportTo(inspectable);
+    const serialized: any = {};
+    manifest.exportTo(serialized);
 
     inspect.styles.name = `cyan`;
 
-    this.context.stdout.write(`${inspect(inspectable, {
+    this.context.stdout.write(`${inspect(serialized, {
       depth: Infinity,
       colors: true,
       compact: false,
     })}\n`);
+
+    const manifestPath = ppath.join(this.context.cwd, Manifest.fileName);
+    await xfs.changeFilePromise(manifestPath, `${JSON.stringify(serialized, null, 2)}\n`);
```

Comment:
Should probably be normalized so it uses `\r\n` on Windows

Reply:
Oh, indeed 👍